src/main/java/com/google/api/generator/gapic/composer/common/AbstractServiceClientTestClassComposer.java

```diff
@@ -472,7 +476,13 @@ private MethodDefinition createRpcTestMethod(
             VariableExpr.withVariable(
                 Variable.builder().setType(methodArg.type()).setName(methodArgName).build());
         argExprs.add(varExpr);
-        Expr valExpr = createDefaultValue(methodArg, resourceNames);
+        Map<String, String> valuePatterns = Collections.emptyMap();
+        if (getTransportContext().useValuePatterns() && method.hasHttpBindings()) {
+          valuePatterns = method.httpBindings().getPathParametersValuePatterns();
```

Comment:
Same operation for all iterations. Move out of the `for` loop to signify that (and avoid unnecessary repetition).

Reply:
moved